### DISCUSSION TITLE
How to install Apache skywalking  Rover in centos machine?

### DISCUSSION THREAD

**Discussion by Mr-XGJ** [apache/skywalking#12205] [Q&A]
My requirement is to install Skywalking Rover on a Linux CentOS system to collect network data. Currently, version 0.0.4 has been installed on the Linux CentOS system, but no data is being collected. Can Skywalking Rover be directly deployed on Linux?

**Comment by wu-sheng** [comment]
Rover 0.6(latest) only supports k8s deployment.

**Reply by Mr-XGJ** [reply]
Thank you. Have previous versions supported deployment on Linux?

**Reply by wu-sheng** [reply]
Yes, it was. But those versions are profiling only, not 7x24 monitoring.

**Reply by Mr-XGJ** [reply]
Ok, I'm a beginner in eBPF. Currently, I've set up Skywalking OAP and Rover 0.4 on a Linux environment, but I haven't found any data analysis. Could you please advise if I'm doing something wrong? Does the official website have any documentation for deploying in a Linux environment?

**Reply by wu-sheng** [reply]
Search for what profiling means. It is not collecting data automatically. You could read and follow this, https://skywalking.apache.org/blog/ebpf-enhanced-http-observability-l7-metrics-and-tracing/

But never use the latest version, only blog mentioned versions support that.

**Reply by Mr-XGJ** [reply]
Thank you very much for your support.
